IMPLEMENTATIONS.md

```diff
@@ -20,6 +20,15 @@ These tools are not necessarily endorsed by the OAI.
 | Tavis.OpenApi | [GitHub/tavis-sofware/Tavis.OpenApi](https://github.com/tavis-software/Tavis.OpenApi/) | dotnet | C# based parser with definition validation and migration support from V2 |
 
 
+#### Editors
+
+| Title          | Project Link | Language |Description                          |
```

Comment:
Possibly more useful would be a `platform` column with e.g. `Eclipse` or `web` in these cases.